### ISSUE TITLE
Library changes

### ISSUE THREAD

**Issue by richsalz** [The-DevX-Initiative/RCIG_Coordination_Repo#59]
This is a companion to #55 

It would be nice to have an `os::get_entropy` call that did the right (FIPS-compatible) thing.  See https://github.com/openssl/openssl/tree/master/providers/implementations/rands/seeding for examples.

**Comment by newpavlov** [comment]
I think the [`getrandom`](https://docs.rs/getrandom/) crate already covers it.

**Comment by richsalz** [comment]
@newpavlov you are right, it does!  I wonder if that should be added to the crypto list?
